miflora-mqtt-daemon.py

```diff
@@ -307,6 +307,15 @@ def flores_to_openhab_items(flores, reporting_mode):
             payload['name'] = "{} {}".format(flora_name, sensor.title())
             if 'device_class' in params:
                 payload['device_class'] = params['device_class']
+            payload['unique_id'] = "{}-{}".format(flora['mac'].lower().replace(":", ""), sensor)
+            payload['device'] = {
+                    'identifiers' : ["MiFlora{}".format(flora['mac'].lower().replace(":", ""))],
+                    'connections' : [["mac", flora['mac'].lower()]],
+                    'manufacturer' : 'Xiaomi',
+                    'name' : flora_name,
+                    'model' : 'HHCCJCY01',
```

Comment:
Is the model the best fit here? https://xiaomi-mi.com/sockets-and-sensors/xiaomi-huahuacaocao-flower-care-smart-monitor/ How about "model: MiFlora Flower Monitor"?

Reply:
HHCCJCY01 is the PN (Product Number) / Model Number of "MiFlora Flower Monitor". This sensor is also sold now by VegTrug under another name (same PN), so that's is why I've chosen PN instead of a descriptive name. Vegtrug also now sells different Flower sensor with PN GCLS002 (outdoor version, named VegTrug Grow Care Garden). My opinion is that PN is best option to describe model, but you are the owner so decide yourself.

Reply:
Good argument with VegTrue. Agree.